lib/pbench/test/unit/server/__init__.py

```diff
@@ -1,3 +1,7 @@
 import pytest
 
 pytest.register_assert_rewrite("pbench.test.unit.server.query_apis.commons")
+
+ADMIN_USER_ID = "6"
+DRB_USER_ID = "3"
+TEST_USER_ID = "20"
```

Comment:
One might question how we arrive at these numbers until someone changes them and the tests start falling apart :laughing:

Reply:
As long as the three numbers are **different** from each other, I _think_ the tests will hold together.  🙂